internal/cmd/gocdk/biome.go

```diff
@@ -28,37 +32,111 @@ func registerBiomeCmd(ctx context.Context, pctx *processContext, rootCmd *cobra.
 		Short: "TODO Manage biomes",
 		Long:  "TODO more about biomes",
 	}
+	var launcher string
 	biomeAddCmd := &cobra.Command{
 		Use:   "add BIOME_NAME",
 		Short: "TODO Add BIOME_NAME",
 		Long:  "TODO more about adding biomes",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
-			return biomeAdd(ctx, pctx, args[0])
+			return biomeAdd(ctx, pctx, args[0], launcher)
 		},
 	}
+	// TODO(rvangent): There should be a way to specify answer to launcher-specific
+	// prompts via flags. Maybe a JSON snippet?
+	biomeAddCmd.Flags().StringVar(&launcher, "launcher", "", "the launcher for the new biome")
 	biomeCmd.AddCommand(biomeAddCmd)
 
-	// TODO(rvangent): More biome subcommands.
+	// TODO(rvangent): More biome subcommands: delete, list, ...?
 
 	rootCmd.AddCommand(biomeCmd)
 }
 
-func biomeAdd(ctx context.Context, pctx *processContext, newName string) error {
-	// TODO(clausti) interpolate launcher from one supplied as a flag
-	pctx.Logf("Adding biome %q...", newName)
+// launchSpecifier is used to pass launch specifier key/value pairs to
+// outputs.tf for a new biome.
+type launchSpecifier struct {
+	Key, Value string
+}
+
+// launcherData is passed as template data to biome.json for a new biome.
+type launcherData struct {
+	Launcher     string
+	ServeEnabled bool
+}
+
+func biomeAdd(ctx context.Context, pctx *processContext, biome, launcher string) error {
+	pctx.Logf("Adding biome %q...", biome)
 
 	moduleRoot, err := pctx.ModuleRoot(ctx)
 	if err != nil {
 		return xerrors.Errorf("biome add: %w", err)
 	}
 
-	// Create a whole new directory, copied from /biome, into biomes/<newName>.
+	if launcher == "" {
+		var err error
+		// TODO(rvangent): Do this as a selection instead of freeform input?
+		reader := bufio.NewReader(pctx.stdin)
+		launcher, err = prompt.String(reader, pctx.stderr, "Please enter a launcher to use (local, cloudrun)", "local")
+		if err != nil {
+			return err
+		}
+	}
+
+	data := &launcherData{Launcher: launcher}
+	var launchSpecifiers []*launchSpecifier
+	var saveActions []*static.Action
+	switch launcher {
+	case "local":
```

Comment:
(Optional) Create named constants to share with the `newLauncher` function?

Reply:
I'm not going to do that here. IMHO there should be a larger refactor where you specify everything for a launcher in one place, and this would just call it.